src/main/java/core/basesyntax/service/FileReaderService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import java.io.File;
+import java.util.List;
+
+public interface FileReaderService {
+    public List<String> getDataFromFile(File file);
```

Comment:
The parameter type should be a String representing the path to the file, rather than a File object. This is to follow the principle of avoiding hardcoding and using relative paths to resources.